src/ip_vs_conn.erl

```diff
@@ -46,10 +47,10 @@ parse(#ip_vs_conn_state{
 parse({Conn,  #ip_vs_conn_status{ tcp_state = TcpState, conn_state = ConnState}}) ->
     parse(#ip_vs_conn_state{connection = Conn, conn_state = ConnState, tcp_state = TcpState}).
 
-file_fold(Func, Z, {ok, Fd}) -> file_fold_line(Func, Z, Fd, file:read_line(Fd));
+file_fold(Func, Z, Fd) -> file_fold_line(Func, Z, Fd, file:read_line(Fd));
 file_fold(_, Z, _) -> Z.
```

Comment:
This can never match now because the first clause no longer has a tuple to differentiate.